draft-ietf-quic-recovery.md

```diff
@@ -1188,9 +1188,10 @@ SetLossDetectionTimer():
     return
 
   // Don't arm timer if there are no ack-eliciting packets
-  // in flight and the handshake is complete.
-  if (endpoint is client with 1-RTT keys &&
-      no ack-eliciting packets in flight):
+  // in flight and the endpoint is a server or has
+  // completed the handshake.
+  if (no ack-eliciting packets in flight &&
+      (endpoint is server or has 1-RTT keys)):
```

Comment:
endpoint is server or has received hs ack or an ack of an 1-RTT packet (not a 1-RTT packet, this could be an ack of 0RTT)

Reply:
Sorry, just add "has received HS ack" and we're good. There's a case where the server flight doesn't fit in a cwnd and the extra text may save us a crypto timeout.